bitnami/superset/templates/beat/deployment.yaml

```diff
@@ -0,0 +1,185 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.beat.enabled }}
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  name: {{ printf "%s-beat" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
```

Comment:
In other charts we create a macro like superset.beat.fullname for simplicity. The same wit the other componens